lib/matplotlib/widgets.py

```diff
@@ -1958,9 +1974,11 @@ class SpanSelector(_SelectorWidget):
     See also: :doc:`/gallery/widgets/span_selector`
     """
 
-    def __init__(self, ax, onselect, direction, minspan=None, useblit=False,
-                 rectprops=None, onmove_callback=None, span_stays=False,
-                 button=None):
+    @_api.rename_parameter("3.5", "span_stays", "interactive")
+    def __init__(self, ax, onselect, direction, minspan=0, useblit=False,
+                 rectprops=None, maxdist=10, marker_props=None,
+                 onmove_callback=None, interactive=False,
+                 button=None, drag_from_anywhere=False):
```

Comment:
I think you're changing the order of the arguments here which is backwards incompatible. Someone may be (inadvisably) using positional args for every param and this will break that.

Can you switch the order so that all new kwargs are added at the end.

Reply:
Done, I have left `interactive` at the same position as `span_stays` since it replace it.